Code/Editor/Util/ModalWindowDismisser.cpp

```diff
@@ -53,12 +53,12 @@ bool ModalWindowDismisser::eventFilter(QObject* object, QEvent* event)
                 {
                     m_windows.push_back(dialog);
                 }
-                if (!m_dissmiss)
+                if (!m_dismiss)
                 {
                     // Closing the window at the same moment is opened leads to crashes and is unstable,
-                    // so do it after a long 1 ms
-                    QTimer::singleShot(1, this, &ModalWindowDismisser::DismissWindows);
-                    m_dissmiss = true;
+                    // so do it after a long 2 ms (needed for Jenkins to realise the dialog is there in some circumstances)
+                    QTimer::singleShot(2, this, &ModalWindowDismisser::DismissWindows);
+                    m_dismiss = true;
```

Comment:
This is not a real fix.
Adding a timer is not fixing the underlying issue here and changing it from 1 to 2 milliseconds doesn't fix any race conditions here

Reply:
I agree. This test for this item is already disabled. We should deep dive to solve the real root cause.

Reply:
I have tried many ways of removing this but can't find a reliable method that works in all cases. The time delay to allow Qt to setup correctly does work. Can you suggest anything else?

Reply:
Rewritten the dismisser to not use a timer.